### ISSUE TITLE
Unable to get new feature to download SOAP API docs to work

### ISSUE THREAD

**Issue by ghgofort** [SqrTT/prophet#238]
**Describe the bug**
When I navigate to the `webreferences` folder and open the context menu when selecting the `.wsdl` file I don't see the option to download the API docs in the menu:
![image](https://user-images.githubusercontent.com/16811151/81087789-1f6c0e00-8ec8-11ea-8061-7c78649b29bd.png)

**To Reproduce**
1. Open VSCode File Explorer and find webreferences folder inside of cartridge with SOAP service definition.
2. Right click on the `.wsdl` extension file to attempt to download the API docs.
3. The expected option "Download SOAP Web Service API..." is missing from the context menu.

**Expected behavior**
I would expect to see the option as described in the README.md.

**Desktop (please complete the following information):**
- OS: macOS Catalina - Version 10.15.4
-  

 - VSCode version - 1.44.2
 - The extension version: 1.3.3

**Comment by SqrTT** [comment]
hi @ghgofort 
thanks for reporting. Yes, that's correct. Feature exist in master branch but not released yet (not published).
I've plan to publish once fix few issues related to javadoc creation (for some reason javadoc don't create docs while eclipse does).

PS. published version have correct readme without SOAP feature. 
https://marketplace.visualstudio.com/items?itemName=SqrTT.prophet

**Comment by ranveer5289** [comment]
hi @SqrTT,

Can you add a bit more detail regarding what exactly doesn't work for javadoc? When I tested this feature it worked perfectly fine. 

I tested #233  on OSX

**Comment by SqrTT** [comment]
@ranveer5289 in my case it fails. Errors is about missing classes definition.

Example from eclipse:
```
     [javadoc] C:\Users\IEUser\Desktop\New folder\Payment\src\webreferences\Payment\ApiType.java:105: error: package org.apache.axis.description does not exist
     [javadoc]     private static org.apache.axis.description.TypeDesc typeDesc =
     [javadoc]                                               ^
     [javadoc] C:\Users\IEUser\Desktop\New folder\Payment\src\webreferences\Payment\ApiType.java:127: error: package org.apache.axis.description does not exist
     [javadoc]     public static org.apache.axis.description.TypeDesc getTypeDesc() {
     [javadoc]                                              ^
     [javadoc] C:\Users\IEUser\Desktop\New folder\Payment\src\webreferences\Payment\ApiType.java:134: error: package org.apache.axis.encoding does not exist
     [javadoc]     public static org.apache.axis.encoding.Serializer getSerializer(
```
but regardless error eclipse creates docs but if run from cli or vscode it doesn't :(

I'm on Linux/Ubuntu. (Eclipse on windows - virtual machine)

**Comment by ranveer5289** [comment]
@SqrTT : is it possible for you to share the WSDL file? I would like to give this a try myself because the WSDL I tested with before also generated a lot of error's but always docs were successfully created.

**Comment by SqrTT** [comment]
sure. I'm playing with [this WSDL](https://pagamentos.reduniq.pt/sandbox-api-gateway/v1.0/soap/index.php?wsdl)

**Comment by ranveer5289** [comment]
Thanks, I tried and indeed no documentation is generated.

I even tried my previous WSDL's which were working fine before when I submitted the PR but now they are also not working. Seems like something changed with vscode.

Because if i try the same `javadoc` implementation from a separate node script it works perfectly fine and docs are generated.

```js
var ch = require('child_process');

process.chdir('/path/to/java/files');
ch.spawnSync('javadoc', ['-d' , 'docs', '*.java'], {stdio:"ignore", shell: true});
```

I will keep on digging and will try to downgrade the vscode version.

**Comment by ranveer5289** [comment]
I got a bit more debugging information. Looks like the issue is related to JDK version.

I can now replicate the issue from javadoc cli as well. If use JDK12 no documentation is generated & if I use JDK8 documentation is properly generated.

**Comment by ranveer5289** [comment]
@SqrTT : 

As per [official documentation](https://documentation.b2c.commercecloud.salesforce.com/DOC1/topic/com.demandware.dochelp/SiteDevelopment/InstallUXStudio.html) SFCC officially supports JDK8 only for UXStudio.

So, when you have some time is it possible for you to confirm that if you use JDK8 functionality works fine at your end as well.

Once confirmed from your end that functionality indeed works, I see 2 ways to handle this feature request

1) I add an additional check in the implementation for java version & if any version other than 1.8 is found just show an error message and also add this implementation behind a feature toggle.

2) We remove this functionality from the plugin as we cannot guarantee that it will always work 100% fine.

I will leave this decision to you :)

**Comment by SqrTT** [comment]
released in 1.3.4.
I've just add warning regarding required version of JDK. Even `javadoc` may not work properly downloaded java files are still useful.

Thank you guys for your contribution
